tests/testthat/helper-outpack.R

```diff
@@ -184,7 +184,7 @@ outpack_packet_end_quietly <- function(...) {
 
 forcibly_truncate_file <- function(path) {
   permissions <- fs::file_info(path)$permissions
-  fs::file_chmod(path, "a+w")
-  file.create(path)
+  fs::file_delete(path)
+  fs::file_create(path)
```

Comment:
Somehow, chmod isn't immediately visible when working over Samba, even after it returns.
I had to do this to work around it and get the tests passing.

```sh
touch hello.txt
chmod 444 hello.txt; chmod 666 hello.txt; ls -l hello.txt
```

Running the second line many times, it will sometimes print the permissions as `-rwxr-xr-x`, sometimes as `-r-xr-xr-x`.